SECURITY.md

```diff
@@ -15,3 +15,4 @@ Please refer to the [Bug Bounty Page](https://hackenproof.com/avalanche) for the
 ## Supported Versions
 
 Please use the [most recently released version](https://github.com/ava-labs/avalanchego/releases/latest) to perform testing and to validate security issues.
+// TODO@ pure avax copy, update with camino or remove
```

Comment:
Will be more important if we finally merge camino-node and caminogo -> I suggest keep / rename

Reply:
thats outdated, was addressed in some of fixes commits